package.json

```diff
@@ -77,7 +77,9 @@
     "docs:acceptApiChanges": "scripts/use_node --max-old-space-size=6144 scripts/check_published_api_changes.js --accept",
     "osd:bootstrap": "scripts/use_node scripts/build_ts_refs && scripts/use_node scripts/register_git_hook",
     "spec_to_console": "scripts/use_node scripts/spec_to_console",
-    "pkg-version": "scripts/use_node -e \"console.log(require('./package.json').version)\""
+    "pkg-version": "scripts/use_node -e \"console.log(require('./package.json').version)\"",
+    "cypress:run-without-security": "env TZ=America/Los_Angeles NO_COLOR=1 cypress run --headless --env SECURITY_ENABLED=false"
```

Comment:
Should we introduce `SECURITY_ENABLED` concept into OSD core as security plugin is a plugin.

Reply:
I think so because these are Cypress env variables that needs to be set in order to override some of the configurations/commands based on if security plugin is installed or not. 
https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/cypress/utils/commands.js#L75

Reply:
Just feel a little bit weird as for OSD core, it has no knowledge on authentication. @kavilla What's your opinion?